bin/wasm-node/rust/src/network_service.rs

```diff
@@ -235,6 +236,22 @@ impl NetworkService {
         result
     }
 
+    /// Sends transaction to the given peer.
+    // TODO: more docs
+    pub async fn announce_transaction(
+        self: Arc<Self>,
+        peers: Vec<PeerId>,
```

Comment:
A `peers` parameter is not really what I had in mind. The network service is supposed to be aware of the peers to send the transaction to.